### DISCUSSION TITLE
Multi-GPU DDP training failed

### DISCUSSION THREAD

**Discussion by Nahid-Ahsan** [coqui-ai/TTS#3576] [General Q&A]
I try to training YourTTS model on custom data in multi gpu single node. It through a error message. But the single gpu training works perfectly. 

As I saw in multi-gpu, the dataloader got the list of indices instead of single integer. What should be the possible solution of this.
![Screenshot 2024-02-14 084653](https://github.com/coqui-ai/TTS/assets/42906020/d54a26d4-8788-40d2-b0cf-86605b63cda2)

**Comment by ysujiang** [comment]
I also encountered the same problem. Have you solved it
